db/db_impl_compaction_flush.cc

```diff
@@ -2074,6 +2076,33 @@ void DBImpl::InstallSuperVersionAndScheduleWork(
       mutable_cf_options.max_write_buffer_number;
 }
 
+// ShouldIPurge is called by FindObsoleteFiles when doing a full scan,
+// and db mutex (mutex_) should already be held. This function performs a
+// linear scan of an vector (files_grabbed_for_purge_) in search of a
+// certain element. We expect FindObsoleteFiles with full scan to occur once
+// every 10 hours, and the size of the vector is small. Therefore, the cost
```

Comment:
maybe mention the 10 hours is the default value to avoid confusion, since it won't be true for everyone

Reply:
Done.